core/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -58,11 +59,14 @@
 
     private final boolean allowExplicitIndex;
     private final IndicesQueriesRegistry indicesQueriesRegistry;
+    private AggregatorParsers aggParsers;
 
 
     @Inject
-    public RestMultiSearchAction(Settings settings, RestController controller, Client client, IndicesQueriesRegistry indicesQueriesRegistry) {
+    public RestMultiSearchAction(Settings settings, RestController controller, Client client, IndicesQueriesRegistry indicesQueriesRegistry,
```

Comment:
not to be fixed in this pr but should aggs use a similar naming to queries? eg. IndicesAggregationsRegistry?